metricq/cli/wrapper.py

```diff
@@ -0,0 +1,136 @@
+import logging
+from typing import Any, Callable, Optional, cast
+
+import click
+import click_log  # type: ignore
+from click import Context, option
+from dotenv import find_dotenv, load_dotenv
+
+from .. import get_logger
+from .params import MetricParam, TemplateStringParam
+from .syslog import SyslogFormatter, get_syslog_handler
+from .types import FC
+
+# We do not interpolate (i.e. replace ${VAR} with corresponding environment variables).
+# That is because we want to be able to interpolate ourselves for metrics and tokens
+# using the same syntax. If it was only ${USER} for the token, we could use the
+# override functionality, but most unfortunately there is no standard environment
+# variable for the hostname. Even $HOST on zsh is not actually part of the environment.
+# ``override=false`` just means that environment variables have priority over the
+# env files.
+load_dotenv(dotenv_path=find_dotenv(".metricq"), interpolate=False, override=False)
+
+
+def metricq_server_option() -> Callable[[FC], FC]:
+    return option(
+        "--server",
+        type=TemplateStringParam(),
+        metavar="URL",
+        required=True,
+        help="MetricQ server URL.",
+    )
+
+
+def metricq_token_option(default: str) -> Callable[[FC], FC]:
+    return option(
+        "--token",
+        type=TemplateStringParam(),
+        metavar="CLIENT_TOKEN",
+        default=default,
+        show_default=True,
+        help="A token to identify this client on the MetricQ network.",
+    )
+
+
+def metricq_syslog_option(required: Optional[bool]) -> Callable[[FC], FC]:
+    def enable_syslog(ctx: Context, param: Any | None, value: Optional[str]) -> None:
+        if required and value is None:
+            raise ValueError("Input syslog is missing")
+        if value is not None:
+            logger = get_logger()
+
+            program_name: str = "metricq_program"
+            metric = ctx.params.get("metric", None)
+
+            if metric is not None:
+                program_name = metric
```

Comment:
```suggestion
            program_name = ctx.params.get("metric", "metricq_program")
```

This would be the same code in one line.